### ISSUE TITLE
Firefox: NVDA table navigation command, unable to move to specific column

### ISSUE THREAD

**Issue by surfer0627** [nvaccess/nvda#13314]
In this page, NVDA table navigation command, unable to move to specific column (from column 5 to column 2).

This case occurs in Firefox. It works as expected in Chrome and Edge.

### Steps to reproduce:

1. Visit [this page](https://gerrit.libreoffice.org/q/status:open).
2. Press t to move to next table.
3. Press control+alt+rightArrow to move to row 1 column 5 (Reviewers).
4. Press control+alt+downArrow to move to row 2 column 5.
5. Press control+alt+leftArrow to move to row 2 column 2 (Subject content).

#### Actual behavior:
The caret moves between column 4 and 5.

#### Expected behavior:
The caret moves to column 2 as expected.

#### NVDA installed/portable/running from source:
Portable

#### NVDA version:
2021.3.1

#### Windows version:
Windows version 10.0.19042

#### Name and version of other software in use when reproducing the issue:
* [Firefox version 96.0.3](http://ftp.mozilla.org/pub/firefox/releases/96.0.3/win64/zh-TW/)

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA 2020.4

#### If addons are disabled, is your problem still occuring?
Yes.

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No.

**Comment by lukaszgo1** [comment]
cc @jcsteh

**Comment by jcsteh** [comment]
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1754352 for part of this. The other part needs https://bugzilla.mozilla.org/show_bug.cgi?id=1686391.

**Comment by Adriani90** [comment]
Te second bug in the comment above seems fixed in cache the world in Firefox 113 enabled by default, not sure about the first bug though.

**Comment by Adriani90** [comment]
In the first firefox bug it is stated this is rather an authoring bug due to custom css layout table used which is not supported in Firefox. Closing as invalid.
